### ISSUE TITLE
Check GTFS data over the years

### ISSUE THREAD

**Issue by kauebraga** [ipeaGIT/acesso_oport#56]
- [ ] Calculate rio access 2017 again with the correct supervia GTFS;
- [ ] Fix rio subway GTFS;
- [ ] Compare 2018 to 2017 GTFSs;
- [ ] Check if Linha 5 and Linha15 are incorporated in spo's GTFS (both 207 and 2019)

**Comment by kauebraga** [comment]
About the sao paulo comparison between 2017 and 2019.

We can see that there wasn't a new subway line in the period, but both the `METRÔ 15` and `METRÔ L5` had an significant expansion in the period. We can also see that the general speed on these lines had a big drop, from 33 to 22 in the `METRÔ 15` and from 32 to 25 in the `METRÔ L5`:


|route_id | length_2017 (km)| length_2019 (km)| n_stops_2017| n_stops_2019| speed_2017 (km/h)| speed_2019 (km/h)|
|:--------|----------------:|----------------:|------------:|------------:|-----------------:|-----------------:|
|METRÔ 15 |            2.228|            8.748|            2|            7|          33.42000|          21.87000|
|METRÔ L1 |           20.248|           20.487|           23|           23|          29.58312|          29.93231|
|METRÔ L2 |           14.619|           15.046|           14|           13|          29.23800|          30.09200|
|METRÔ L3 |           22.977|           22.608|           18|           18|          25.60904|          25.19777|
|METRÔ L4 |            9.017|           11.598|            7|           10|          38.64429|          33.13714|
|METRÔ L5 |            9.562|           19.921|            7|           17|          31.87333|          24.90125|

When we compare accessibility (opportunities := 1) between 2017 and 2019, as log(access_2019/acceso_2017):

- São Paulo as a whole
![comp_2017_2019_spo](https://user-images.githubusercontent.com/22420248/96293576-5be87d80-0fc1-11eb-9c79-389ec03c955a.png)

- Zooming in the L5 region
![comp_2017_2019_spo_l5](https://user-images.githubusercontent.com/22420248/96293606-673ba900-0fc1-11eb-8825-ed62f5fa0ff9.png)

- Zooming in the L15 region
![comp_2017_2019_spo_l15](https://user-images.githubusercontent.com/22420248/96293630-71f63e00-0fc1-11eb-8829-a9e1d5bbaea1.png)

**Comment by kauebraga** [comment]
It seems like the expansions parts weren't fast enough to promote a higher level of accessibility locally

**Comment by rafapereirabr** [comment]
So despite of the network expansion, there were no accessibility gains because of low speed in the new rail corridors.... Hum, the low speeds seem a bit strange. I'll check this with Giannotti and Arbex.

**Comment by kauebraga** [comment]
Yes this seems like a GTFS issue.. they repeated the travel times from the previous segments, it just happened that the new lengths were shorter than the previous ones
